src/lib/db/feature-toggle-store.ts

```diff
@@ -171,34 +173,40 @@ export default class FeatureToggleStore implements IFeatureToggleStore {
 
     async setLastSeen(data: LastSeenInput[]): Promise<void> {
```

Comment:
We should move the setLastSeen out of this store. We already have a separate store updating the `client_metrics_env` which is mostly the same challenges (`ClientMetricsStoreV2`)